src/components/slider/slider.ts

```diff
@@ -264,6 +351,30 @@ export class SliderRenderer {
   addFocus() {
     this._sliderElement.focus();
   }
+
+  /**
+   * Draws ticks onto the tick container.
+   * @param tickSeparation How far apart to draw the ticks.
+   */
+  drawTicks(tickSeparation: number) {
+    let tickContainer = <HTMLElement>this._sliderElement.querySelector('.md-slider-tick-container');
+    let tickContainerWidth = tickContainer.getBoundingClientRect().width;
+    let lastTickContainer =
+        <HTMLElement>this._sliderElement.querySelector('.md-slider-last-tick-container');
+    // Subtract 1 from the tick separation to center the tick.
+    // TODO: Perf test this.
+    tickContainer.style.background = `repeating-linear-gradient(to right, #000000, #000000 2px,
```

Comment:
`#000000` -> `black`

This is okay for this PR, but we also need to consider whether this should be a different color for dark themes.